### PR TITLE
Update Bitcoin Core project page

### PR DESCRIPTION
**Pull request by dergigi**
Rewrites the `Bitcoin Core` project page to match the newer structure and sharpen the funding context.

- Explains what `Bitcoin Core` is, why OpenSats funds work around it, and what maintenance support still matters
- Tracks https://github.com/OpenSats/content/issues/19

---

Build preview:
- [/projects/bitcoin-core](https://os-website-git-content-update-bitcoin-core-description-opensats.vercel.app/projects/bitcoin-core)

### CODE REVIEW

**Comment by Kokkomaki** [comment]
unable to access  https://vercel.com/opensats/os-website/3tRmz1i1ZZB3X7VMi1UtD9RbGAsa

File: data/projects/bitcoin-core.mdx [hunk 1]
```diff
@@ -1,21 +1,57 @@
...
-## About this project
+Bitcoin Core is the full node software that validates blocks and transactions, enforces consensus rules, relays data across the peer-to-peer network, and lets users verify Bitcoin for themselves.
 
-Bitcoin uses peer-to-peer technology to operate with no central authority or
-banks; managing transactions and the issuing of bitcoins is carried out
-collectively by the network. Bitcoin is open-source; its design is public,
-nobody owns or controls Bitcoin and everyone can take part. Through many of its
-unique properties, Bitcoin allows exciting uses that could not be covered by any
-previous payment system.
+OpenSats began funding Bitcoin Core contributors in 2023 through its [LTS program][lts-announce]. LTS grantees include Marco Falke, Josie Baker, Sjors Provoost, Vasil Dimov, Gleb Naumenko, Furszy, Will Clark, 0xB10C, Bruno Garcia, Jon Atack, and Andrew Toth. Additional contributors joined through [Caring for Core][caring], [More for Core][more-for-core], and [5 Grants to Strengthen Bitcoin Development][five-grants]: tdb3, David Gumberg, Hodlinator, L0rinc, kevkevinpal, Daniela Brozzoni, janb84, Naiyoma, and Daniel Pfeifer.
```
Comment (Kokkomaki) [inline data/projects/bitcoin-core.mdx:18]: Does it matter which order these people are mentioned? Is alphabetic order okay, or do we go with chronological, or something else? Thinking in terms of future writing, too.

File: data/projects/bitcoin-core.mdx [hunk 2]
```diff
@@ -1,21 +1,57 @@
...
+
+## Why fund it?
+
+Fewer than 50 people regularly contribute code to software underpinning a network worth over a trillion dollars. For comparison, Meta employs around 20,000 engineers for a comparable market cap. The total annual budget for all Bitcoin Core development organizations combined is around $8 million.
+
+Most of that budget goes toward maintenance. Bug fixes, test coverage, CI upkeep, build system work, release engineering, edge cases that show up in the wild. A [widely cited estimate][maintenance-source] puts maintenance at well over 70% of all project activity. The protocol upgrades get attention; the day-to-day work does not.
```
Comment (Kokkomaki) [inline data/projects/bitcoin-core.mdx]:
```suggestion
Most of that budget goes toward maintenance. This includes, for example bug fixes, test coverage, CI upkeep, build system work, release engineering, and even edge cases that show up in the wild. A [widely cited estimate][maintenance-source] puts maintenance at well over two thirds of all project activity. In other words, the protocol upgrades get more attention while the day-to-day work does not.
```

File: data/projects/bitcoin-core.mdx (hunk 1 above)
Comment (dergigi) [reply data/projects/bitcoin-core.mdx:18]: I think chronological makes the most sense.

**Review by Arvin21M — APPROVED** [review]
- Reads clearly
- All links and buttons work and point to the right place
- Looks good to go
- :shipit:
